### DISCUSSION TITLE
v2.69.0

### DISCUSSION THREAD

**Discussion by nunocoracao** [nunocoracao/blowfish#1505] [Announcements]
### TL;DR
Add `enableEmoji = true` to your `hugo` or `config` files to retain existing behavior. 

### Explanation
This release introduces a breaking change with https://github.com/nunocoracao/blowfish/pull/1504
Blowfish now supports emoji control at the Hugo level instead of the template level. The above PR submitted by @racehd removed the hardcoded `emojify` calls but it requires changes to the config files to support the old behavior. This means that to retain the previous emojis feature you will need to add `enableEmoji = true` to your `hugo` or `config` files. 

## What's Changed
* ⚙️ Update Hugo supported version by @github-actions in https://github.com/nunocoracao/blowfish/pull/1503
* ✏️ Fix typo in docs/partials/index.md by @racehd in https://github.com/nunocoracao/blowfish/pull/1496
* 📌 Bump puppeteer from 22.9.0 to 22.10.0 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1500
* 📌 Bump @headlessui/react from 2.0.3 to 2.0.4 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1501
* ✨ Use markdown emojify instead of dump emojify by @zoriya in https://github.com/nunocoracao/blowfish/pull/1504
* 🔖 v2.69.0 by @nunocoracao in https://github.com/nunocoracao/blowfish/pull/1494

## New Contributors
* @racehd made their first contribution in https://github.com/nunocoracao/blowfish/pull/1496
* @zoriya made their first contribution in https://github.com/nunocoracao/blowfish/pull/1504

**Full Changelog**: https://github.com/nunocoracao/blowfish/compare/v2.68.0...v2.69.0

<hr /><em>This discussion was created from the release <a href='https://github.com/nunocoracao/blowfish/releases/tag/v2.69.0'>v2.69.0</a>.</em>

**Comment by racehd** [comment]
> Explanation
This release introduces a breaking change with https://github.com/nunocoracao/blowfish/pull/1496
Blowfish now supports emoji control at the Hugo level instead of the template level. The above PR submitted by @racehd

Wrong PR, should be #1504  and credit to @zoriya and not me. don't want to steal credit! Thanks!

**Reply by nunocoracao** [reply]
Thanks @racehd updated... my bad
sorry @zoriya
